attributioncode/validator.go

```diff
@@ -50,9 +59,9 @@ func (v *Validator) Validate(code, sig string) (string, error) {
 	}
 
 	logEntry = logrus.WithField("code", unEscapedCode)
-	if len(unEscapedCode) > 200 {
-		logEntry.WithField("code_len", len(code)).Error("code longer than 200 characters")
-		return "", errors.New("code longer than 200 characters")
+	if len(unEscapedCode) > 400 {
```

Comment:
We've talked about increasing this to e.g. 600 or 800 to make sure we can accommodate a long UA string if needed.

Reply:
Making a length increase should be fine long term as long as we keep in mind there are existing Firefox releases that reject attribution over 200 characters: https://hg.mozilla.org/releases/mozilla-release/file/2c8be14a89c92e56dbfc10ad32284e628a517a93/browser/components/attribution/AttributionCode.jsm#l24

Reply:
I thought the size limitation had something to do with the stub installer.

Reply:
I have a patch that increases the length firefox accepts: https://phabricator.services.mozilla.com/D57906